src/components/CheckList/CheckListItemPanel.tsx

```diff
@@ -57,7 +59,9 @@ export const CheckListItemPanel: FunctionComponent<ICheckList> = (props) => {
         key: "markComplete",
         text: "Mark Complete",
         iconProps: { iconName: "CheckMark" },
-        disabled: compProps.item.CompletedBy ? true : false,
+        disabled: compProps.item.CompletedBy
+          ? true
+          : false || !compProps.roles?.includes(compProps.item.Lead),
```

Comment:
Good catch.  I actually just removed the whole || expression, because as we made it hide the entire CommandBar instead of disable button, the logic on line 82 overrides this.